packages/fluentui/docs-components/src/CodeSnippet/formatCode.ts

```diff
@@ -1,7 +1,7 @@
 import * as _ from 'lodash';
 import * as _prettier from 'prettier/standalone';
 
-import * as babylon from 'prettier/parser-babylon';
+import * as babel from 'prettier/parser-babel';
```

Comment:
Per [this blog post](https://prettier.io/blog/2019/01/20/1.16.0.html#rename-babylon-parser-to-babel-5647-by-wuweiweiwu) this parser name changed in 1.16, and was removed (at least from the types) in 2.

Reply:
Yes, this correct, thanks 👍